include/clang/StaticAnalyzer/Core/AnalyzerOptions.h

```diff
@@ -679,6 +682,13 @@ class AnalyzerOptions : public RefCountedBase<AnalyzerOptions> {
   /// to false when unset.
   bool shouldDisplayNotesAsEvents();
 
+  /// Returns true if \p InternalInfoName is enabled. If it is, it will be
+  /// displayed in the plist output.
+  ///
+  /// This is controlled by the 'internal-info' string option, which defaults
+  /// to "" (nothing is enabled).
+  bool isInternalInfoEnabled(StringRef InternalInfoName);
```

Comment:
String options in general can be iffy. A FIXME for better configuration of this implementation should be added, and later revised (boolean flags and whatnot perhaps).